### DISCUSSION TITLE
make one for

### DISCUSSION THREAD

**Discussion by DeviateBacon3** [ThioJoe/YT-Spammer-Purge#131] [Ideas]
for your email but it delete your  scam

**Comment by KendallDoesCoding** [comment]
> for your email but it delete your scam

But, on email you dont really get much scam mails, unless you go looking for them. Like, scambaiters obviously do coz they sign up for stuff. I personally, don't.  But, yeah this would still be cool regardless. But, he needs to figure out the gmail api, and for it to be efficient, he needs to actually figure out all the mail app apis. So yeah, youtube gets lots and lots of spam comments, especially for the big youtubers and they have to select one by one, also on email, you just type one of the spam email keywords, like whatsapp, and CTRL+A, tap the delete button, so again it doesnt make sense, but good concept, regardless

**Comment by TechStudent10** [comment]
Um, the title is "YouTube Spammer Purge" not "Spammer Purge". Making one an email client like Gmail, Outlook (yes I use Outlook but like 0.5% of the time) or even iCloud Mail (okay, seriously who uses that?!) won't fit within what this project is about, sorry.

**Reply by KendallDoesCoding** [reply]
> Um, the title is "YouTube Spammer Purge" not "Spammer Purge". Making one an email client like Gmail, Outlook (yes I use Outlook but like 0.5% of the time) or even iCloud Mail (okay, seriously who uses that?!) won't fit within what this project is about, sorry.

Yeah exactly, As I mentioned, its a good concept, if he wants to make it a separate program, but why go to so much trouble, considering its email and much easier to delete spam, for god's sake, spam even comes in a separate folder in your emails

**Reply by TechStudent10** [reply]
EXACTLY. THE SPAM TAB.
